src/components/progress-bar/progress-bar-styles.jsx

```diff
@@ -146,45 +123,23 @@ export default theme => {
 
     primary: {
       ...variantModifierFn('primary', {
-        background: palette.background.secondary,
-        backgroundDisabled: palette.background.muted,
-        borderColor: palette.background.secondary,
-        color: palette.shades.dark.text.default,
-        colorDisabled: palette.action.disabled,
+        background: palette.gray7,
```

Comment:
The colors for this segment L126-130 doesn't follow the same pattern as before. Is this intended?

Reply:
we spread the previous action one as default to minimize the code changes, there is no "grey" default color, same for buttons the first one is a CTA blue.